src/index.js

```diff
@@ -55,15 +55,18 @@ class ServerlessPluginWebpack {
     // Include bundle at function level
     this.serverless.service.functions = service.setFnsPackage(this.serverless.service.functions);
 
+    const runWebpack = this.custom.series ? wpack.runSeries : wpack.run;
+
     // Run webpack
-    return wpack.run(
+    return runWebpack(
       wpack.createConfigs(
         this.originalFunctions,
         webpackConfig,
         this.originalServicePath,
         webpackDefaultOutput,
         webpackFolder
-      )
+      ),
+      this.serverless
```

Comment:
I would prefer not to pass the serverless instance here, otherwise we don't have the same type of logging for series and parallel